src/components/VmDisks/utils.js

```diff
@@ -0,0 +1,21 @@
+
+import { locale as appLocale, DEFAULT_LOCALE } from '../../intl'
+
+function localeCompare (a, b, locale = appLocale) {
+  return a.localeCompare(b, [ locale, DEFAULT_LOCALE ], { numeric: true })
```

Comment:
I did notice that _locale__ coming from intl.js will fallback to DEFAULT_LOCALE. However, the sort function accepts a locale as a parameter, primarily for testing purposes.  When the locale passed in isn't correct, and there isn't a fallback given the behavior is inconsistent across browsers.

In short, it is there to protect against edge cases if the function is ever used elsewhere.

Reply:
* There is no test for invalid locale code so reproducibility of test isn't in risk.
* Current production code won't provide wrong locale code.
* Inconsistent sorting in case of potential future usage and wrong language code isn't much of an issue.

... which makes me find the second item of locales array redundant.

Reply:
I did a bit more reading and it is redundant and doesn't protect against bad locales passed in via function argument.  I'm ok with letting the locale functions throw the errors if the locale isn't correct.  I'll adjust accordingly.

Sorry for my confusion, it's been awhile since I dealt with the ECMA Intl APIs.